site/docs/use-cases/_shared/_convert_model.mdx

```diff
@@ -0,0 +1,3 @@
+## Convert and Optimize Model
+
+Refer to the [Model Preparation](/docs/category/model-preparation) guide for detailed instructions on how to download, convert and optimize models for OpenVINO GenAI.
```

Comment:
Not for this PR. Just wanted to let you know. Clicking a link is a bit hard. At least it's harder than just seeing a command. Maybe its worth providing an example command in such sections asking to follow `Model Preparation` if details are needed.

Reply:
Added